test/TesterInternal/MembershipTests/MembershipTableTestsBase.cs

```diff
@@ -211,12 +232,15 @@ protected virtual string GetAdoInvariant()
             Assert.NotNull(membershipEntry);
         }
 
-        protected async Task MembershipTable_UpdateRow()
+        protected async Task MembershipTable_UpdateRow(bool extendedProtocol = true)
         {
             var tableData = await membershipTable.ReadAll();
-
             Assert.NotNull(tableData.Version);
-            Assert.Equal(0, tableData.Version.Version);
+
+            if (extendedProtocol)
+            {
+                Assert.Equal(0, tableData.Version.Version);
```

Comment:
Currently it wouldn't matter, as Orleans doesn't use the number returned in the version, other than to increment. It can be good as an indicator of sanity.